src/components/TestDeeplinkHandler.tsx

```diff
@@ -21,6 +22,7 @@ export function TestDeeplinkHandler() {
 
       switch (action) {
         case 'import':
+          await savePIN('1111');
```

Comment:
I noticed that in e2e tests the PIN screen would always ask to create a PIN since we never saved it when using the deep link handler here.